### PR TITLE
[statefs-providers] Add Cellular.CallCount. Contributes to JB#32141

### PR DESCRIPTION
**Pull request by martinjones**
Simplifies lipstick ongoing call indication, for example.

### CODE REVIEW

File: src/ofono/provider_ofono.cpp [hunk 1]
```diff
@@ -957,6 +977,50 @@ void Bridge::setup_sim(QString const &path)
         setup_stk(modem_path_);
 }
 
+void Bridge::setup_callManager(QString const &path)
+{
+    qDebug() << "Get call properties";
+    auto update = [this](QString const &n, QVariant const &v) {
+        DBG() << "call prop: " << n << "=" << v;
```
Comment (chriadam) [inline src/ofono/provider_ofono.cpp]: why DBG instead of qDebug?

Comment (deztructor) [inline src/ofono/provider_ofono.cpp]: DBG is compiled in only on debug builds, while qDebug() inlines quite a lot of code

File: src/ofono/provider_ofono.cpp [hunk 2]
```diff
@@ -957,6 +977,50 @@ void Bridge::setup_sim(QString const &path)
         setup_stk(modem_path_);
 }
 
+void Bridge::setup_callManager(QString const &path)
+{
+    qDebug() << "Get call properties";
+    auto update = [this](QString const &n, QVariant const &v) {
```
Comment (deztructor) [inline src/ofono/provider_ofono.cpp]: ```update``` does not do anything, so maybe ```callManager_->GetProperties()```, ```VoiceCallManager::PropertyChanged()``` connection should be removed.
Or some property should be exposed?

Comment (martinjones) [inline src/ofono/provider_ofono.cpp]: The only property is EmergencyNumbers. I'll remove the unneeded code.

**Comment by deztructor** [comment]
LGTM (except some unneeded code in setup_callManager())

**Comment by martinjones** [comment]
@deztructor if you're happy with the change could you merge it please.

**Comment by deztructor** [comment]
@martinjones Sorry for the late merge: for some reason github did not send notification about added commits

**Pull request merged**